src/dmd/dscope.d

```diff
@@ -112,11 +110,16 @@ enum SCOPE
     compile       = 0x0100,   /// inside __traits(compile)
     ignoresymbolvisibility    = 0x0200,   /// ignore symbol visibility
                                           /// https://issues.dlang.org/show_bug.cgi?id=15907
+    onlysafeaccess = 0x0400,  /// unsafe access is not allowed for @safe code
```

Comment:
Nit: Shouldn't this be named onlyunsafeaccess, onlysystemaccess or nosafeaccess as it prevents @safe access but allows unsafe access?

Reply:
Yeah, that's going to bite me later, I fear.

Reply:
https://github.com/dlang/dmd/pull/8041